hadoop-hdds/tools/src/main/java/org/apache/hadoop/hdds/scm/cli/datanode/ListInfoSubcommand.java

```diff
@@ -46,8 +46,9 @@ public class ListInfoSubcommand extends ScmSubcommand {
       defaultValue = "")
   private String ipaddress;
 
-  @CommandLine.Option(names = {"--id"},
-      description = "Show info by datanode UUID.",
+  @CommandLine.Option(names = {"--id", "--uuid"},
+      description = "Show info by datanode UUID. Command line option --id " +
+          "will be removed in later versions.",
```

Comment:
Hi @adoroszlai Thank you for reviewing the changes. Will drop the comments from the PR changeset. But it seems we have some existing Command options where such comments do exist . Do i need to remove the comment from everywhere in ozone?

Reply:
> it seems we have some existing Command options where such comments do exist . Do i need to remove the comment from everywhere in ozone?

For this PR I think it's enough to avoid adding new ones.  We can file a cleanup task for the existing ones.

Reply:
sure thing . I have created https://issues.apache.org/jira/browse/HDDS-7932 for cleanup